metabase/metabase
# Simplify editing question on data app pages

Closes #26457

Adds an ellipsis menu to the top right corner of each data app page card (not for actions; only outside of the editing mode). From the menu, it's now possible to remove a card from a page or tweak its data. When "tweak data" is clicked, a user will be navigated to the query builder with a `?from=/a/:appId/page/:pageId` query parameter. Metabase would show a "Back to app" button next to the logo in the top left + it won't prompt anything when a user is going to save an update to their question. That should make it a bit easier to patch up questions in a data app.

### To Verify

1. New > App > Pick a few tables / models > Create
2. Click the ellipsis icon in the top right, try out "Remove" on something that's not important
3. Click "Tweak data" eventually
4. Ensure you're navigated to the query builder
5. Click the `< {{ DataAppName }}` button next to the Metabase logo in the top left
6. Make sure you go back to the page you were working with
7. Click "Tweak data" again
8. Modify a question query in any way
9. Click "Save"
10. Make sure you were immediately navigated back to the app
11. Make sure the changes to the query that you've made are now reflected in the data app

### Demo

https://user-images.githubusercontent.com/17258145/203633482-db89926d-412a-4a61-abf6-0ce3d02bcb7c.mp4
